proto/ostracon/abci/types.proto

```diff
@@ -44,13 +45,14 @@ message Request {
 
 message RequestBeginBlock {
   bytes                          hash                    = 1;
-  ostracon.types.Header          header                  = 2 [(gogoproto.nullable) = false];
+  tendermint.types.Header        header                  = 2 [(gogoproto.nullable) = false];
   tendermint.abci.LastCommitInfo last_commit_info        = 3 [(gogoproto.nullable) = false];
   repeated tendermint.abci.Evidence byzantine_validators = 4 [(gogoproto.nullable) = false];
+  ostracon.types.Entropy            entropy              = 5 [(gogoproto.nullable) = false];
```

Comment:
How about add below comments above this line?
`// *** Ostracon Extended Fields *** `

Reply:
Good point. and please use the number 1000 because of avoiding conflict with the number of Tendermint.